### ISSUE TITLE
Survival analysis

### ISSUE THREAD

**Issue by gfrc11752** [Huh/PopR_SDGFP#8]
Any chance I could have you throw a check box in the survival analysis tab that will compute an annual survival (0-12 months) for the fawns? I would like to compare my previous MARK estimates (annual) to the model. If its to big of a headache don't worry about it. Thanks!

**Comment by Huh** [comment]
Yes, I can derive that, but for the moment try taking monthly ^ 12 to get an estimate. Josh Nowak 


     On Wednesday, July 15, 2015 2:56 PM, gfrc11752 <notifications@github.com> wrote:
   

 Any chance I could have you throw a check box in the survival analysis tab that will compute an annual survival (0-12 months) for the fawns? I would like to compare my previous MARK estimates (annual) to the model. If its to big of a headache don't worry about it. Thanks!—
